### ISSUE TITLE
Update to taskcluster 5

### ISSUE THREAD

**Issue by marco-c** [mozilla/release-services#1611]
When we update to taskcluster 5.0.0:
```
Traceback (most recent call last):
  File "/tmp/tmp.yc6m4xhYgn/bin/static-analysis-bot", line 11, in <module>
    load_entry_point('static-analysis-bot', 'console_scripts', 'static-analysis-bot')()
  File "/nix/store/na19smsgvvv178rg64jsr92y225ygrdj-python3.6-Click-7.0/lib/python3.6/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/nix/store/na19smsgvvv178rg64jsr92y225ygrdj-python3.6-Click-7.0/lib/python3.6/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/nix/store/na19smsgvvv178rg64jsr92y225ygrdj-python3.6-Click-7.0/lib/python3.6/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/nix/store/na19smsgvvv178rg64jsr92y225ygrdj-python3.6-Click-7.0/lib/python3.6/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/nix/store/08mz8ljhlx23v7ijixacfm7ynvdqf9ny-python3.6-python3.6-mozilla-cli-common-1.0.0/lib/python3.6/site-packages/cli_common/cli.py", line 35, in wrapper
    return func(*args, **kwargs)
  File "/nix/store/pz8nl4cnppd3p1cykshcb40mqxjd5yvy-python3-3.6.5/lib/python3.6/contextlib.py", line 52, in inner
    return func(*args, **kwds)
  File "/app/src/staticanalysis/bot/static_analysis_bot/cli.py", line 64, in main
    taskcluster_access_token=taskcluster_access_token,
  File "/nix/store/08mz8ljhlx23v7ijixacfm7ynvdqf9ny-python3.6-python3.6-mozilla-cli-common-1.0.0/lib/python3.6/site-packages/cli_common/taskcluster.py", line 139, in get_secrets
    taskcluster_access_token,
  File "/nix/store/08mz8ljhlx23v7ijixacfm7ynvdqf9ny-python3.6-python3.6-mozilla-cli-common-1.0.0/lib/python3.6/site-packages/cli_common/taskcluster.py", line 109, in get_service
    return getattr(taskcluster, service_name.capitalize())(options)
  File "/nix/store/hb4dafl5h2rfhvs85k2vppz577hkswxm-python3.6-python3.6-taskcluster-5.0.0/lib/python3.6/site-packages/taskcluster/client.py", line 64, in __init__
    raise exceptions.TaskclusterFailure('rootUrl option is required')
taskcluster.exceptions.TaskclusterFailure: rootUrl option is required
```

We need to fix this and remove https://github.com/mozilla/release-services/pull/1606.

**Comment by MadinaB** [comment]
Facing same issue

**Comment by garbas** [comment]
this was fixed in #1670
